docs/topics/rn-new-features.adoc

```diff
@@ -8,35 +8,31 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-[IMPORTANT]
-====
-Rules for migrating from Java EE8 to Jakarta EE9 and OpenRewrite support are both provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-
-See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
-====
-
-.Rules for migrating from Java EE8 to Jakarta EE9
-{ProductShortName} rules replace Java EE8 artifact dependencies and package import statements and rename XML schema namespaces, prefixed properties, and bootstrapping files for Jakarta EE9. You run these rules by specifying the `jakarta-ee` target.
+.Analyze Known Libraries
+You can now use the `analyzeKnownLibraries` advanced option to analyze libraries embedded within your applications. By default, MTA's default behavior is to only analyze application code, rather than libraries that are declared as dependencies. This option extends your ability to analyze your applications.
 
-.OpenRewrite recipe support
+.Rules for migrating from OpenJDK 8 to OpenJDK 11
+MTA now includes rules to help you migrate applications from OpenJDK 8 (Java 8) to 11.
 
-link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. You can run OpenRewrite recipes by using the link:{ProductDocUserGuideURL}[{ProductShortName} CLI].
+.Technology Reports
+The technology reports have been enriched. New technology classifications have been introduced, many new technologies are now identified and new technology tags have been generated.
 
-The recipe `org.jboss.windup.JavaxToJakarta` renames imported `javax` packages to their `jakarta` equivalents.
+.Apply all Quick Fixes
+In the the MTA IDE plugins and extensions, you can now apply all suggested Quick Fixes at once, rather than one-by-one.
 
```

Comment:
@RichardHoch 
one by one
Is there a reason to capitalize 'Quick Fixes'?

Reply:
You're right about both. Fixed.